### PR TITLE
Change 'modules' to 'libraries' in docs only.

### PR DESCRIPTION
**Pull request by jpmedley**
R: @jeffposnick @addyosmani @gauntface

### CODE REVIEW

**Comment by jpmedley** [comment]
I didn't change anything in packages/ because I want to consult with somebody before touching anything to make sure I'm doing the right thing.

For example, I've looked only in src/ directories but I don't know if I can change `@memberof module:sw-broadcast-cache-update` to `@memberof library:sw-broadcast-cache-update`.

File: docs/contributing.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ accept your pull requests.
 ## Building Projects
 
 There are a couple of commands that will help you make working with these
-modules a little easier:
+libraries a little easier:
```
Comment (addyosmani) [inline docs/contributing.md]: I think `libraries` is more beginner friendly, but modules is more correct. Below we have a number of packages which are technically libraries and then other things that are very much not (e.g `sw-cli` is a CLI). 

I'd be in favor of renaming things currently called modules to `libraries` if we aren't doing it for packages that blur those lines (sw-build) or are outright not libraries (cli).

Comment (gauntface) [inline docs/contributing.md]: +1 This was my only fear.

sw-build and sw-cli will **have** to be downloaded and uses as NPM modules. Everything else is a SW library.

File: docs/sw-lib.md [hunk 2]
```diff
@@ -44,15 +44,15 @@ service worker script, and minimal, straight-forward configuration.
    **Note:** If you use a minifier on your service worker script, be aware that
    Workbox requires one that is ES2015-aware. At the time of Workbox's first
    release (May 2017)
-   [Babili](https://github.com/babel/babili) is the only one that is.
+   [Babili](https://github.com/babel/babili) is the only one that isf.
```
Comment (gauntface) [inline docs/sw-lib.md]: 'isf' a typo here?

File: docs/sw-cli.md [hunk 3]
```diff
@@ -112,9 +112,9 @@ Otherwise maybe you'll want to explore one of the following.
 ### Generating SW in a Build Process
 
 If you have a build process, you might want to use the
-`sw-build` module instead of `sw-cli`.
+`sw-build` library instead of `sw-cli`.
 
-The `sw-build` module allows you to generate the service worker  
+The `sw-build` library allows you to generate the service worker
```
Comment (gauntface) [inline docs/sw-cli.md]: module.

File: docs/sw-cli.md [hunk 4]
```diff
@@ -112,9 +112,9 @@ Otherwise maybe you'll want to explore one of the following.
 ### Generating SW in a Build Process
 
 If you have a build process, you might want to use the
-`sw-build` module instead of `sw-cli`.
+`sw-build` library instead of `sw-cli`.
```
Comment (gauntface) [inline docs/sw-cli.md]: module

File: docs/sw-cli.md [hunk 5]
```diff
@@ -6,13 +6,13 @@ navigation_weight: 1
...
-Get a precaching service worker in about two minutes with the `sw-cli` module.
-Simply install the module then run it from a command line.
+Get a precaching service worker in about two minutes with the `sw-cli` library.
+Simply install the library then run it from a command line.
 
 ## Install
 
 1. [Install Node.js](https://nodejs.org/en/).
-2. Install the module with NPM.
+2. Install the library with NPM.
```
Comment (gauntface) [inline docs/sw-cli.md]: module

File: docs/sw-cli.md [hunk 6]
```diff
@@ -6,13 +6,13 @@ navigation_weight: 1
 
 # sw-cli
 
-Get a precaching service worker in about two minutes with the `sw-cli` module.
-Simply install the module then run it from a command line.
+Get a precaching service worker in about two minutes with the `sw-cli` library.
+Simply install the library then run it from a command line.
```
Comment (gauntface) [inline docs/sw-cli.md]: module

File: docs/sw-cli.md [hunk 7]
```diff
@@ -6,13 +6,13 @@ navigation_weight: 1
 
 # sw-cli
 
-Get a precaching service worker in about two minutes with the `sw-cli` module.
-Simply install the module then run it from a command line.
+Get a precaching service worker in about two minutes with the `sw-cli` library.
```
Comment (gauntface) [inline docs/sw-cli.md]: module

File: docs/sw-build.md [hunk 8]
```diff
@@ -6,14 +6,14 @@ navigation_weight: 2
...
-Simply install the module, then cut and paste one of the provided code samples
+Get a precaching service worker in about ten minutes with the `sw-build` library.
+Simply install the library, then cut and paste one of the provided code samples
 into your build script.
 
 ## Install
 
 1. [Install Node.js](https://nodejs.org/en/).
-1. Install the module with NPM.
+1. Install the library with NPM.
```
Comment (gauntface) [inline docs/sw-build.md]: module

File: docs/sw-build.md [hunk 9]
```diff
@@ -6,14 +6,14 @@ navigation_weight: 2
 
 # sw-build
 
-Get a precaching service worker in about ten minutes with the `sw-build` module.
-Simply install the module, then cut and paste one of the provided code samples
+Get a precaching service worker in about ten minutes with the `sw-build` library.
+Simply install the library, then cut and paste one of the provided code samples
```
Comment (gauntface) [inline docs/sw-build.md]: module

File: docs/sw-build.md [hunk 10]
```diff
@@ -6,14 +6,14 @@ navigation_weight: 2
 
 # sw-build
 
-Get a precaching service worker in about ten minutes with the `sw-build` module.
-Simply install the module, then cut and paste one of the provided code samples
+Get a precaching service worker in about ten minutes with the `sw-build` library.
```
Comment (gauntface) [inline docs/sw-build.md]: module

**Pull request closed without merging**